strongSwan: Modify IPsec configuration

This patch modifies IPsec configuration for X10.

From: Bryce_Chen <Bryce1_Chen@pegatroncorp.com>

--- a/src/starter/ipsec.conf
+++ b/src/starter/ipsec.conf
@@ -4,9 +4,36 @@
 
 config setup
 	# strictcrlpolicy=yes
-	# uniqueids = no
+	uniqueids=never
 
 # Add connections here.
+conn %default
+	left=%defaultroute
+	right=%any
+	leftid=10.0.0.0
+	ike=aes256-sha256-modp2048,aes256-sha1-modp2048,aes256-sha1-modp1024!
+	esp=aes256-sha1
+
+# For WIN 7 or new L2TP
+conn roadwarrior-L2TP-PSK-NAT
+	rightsubnet=10.0.0.0/8,192.168.0.0/16,172.16.0.0/12
+	also=roadwarrior-L2TP-PSK-noNAT
+
+conn roadwarrior-L2TP-PSK-noNAT
+	authby=secret
+	#pfs=no
+	auto=add
+	keyingtries=3
+	rekey=no
+	ikelifetime=8h
+	keylife=1h
+	type=transport
+	leftprotoport=17/1701
+	right=%any
+	rightprotoport=17/%any
+	dpddelay=10
+	dpdtimeout=90
+	dpdaction=clear
 
 # Sample VPN connections
 
--- a/conf/strongswan.conf
+++ b/conf/strongswan.conf
@@ -6,6 +6,7 @@
 
 charon {
 	load_modular = yes
+	duplicheck.enable = no
 	plugins {
 		include strongswan.d/charon/*.conf
 	}
 }
